packages/medusa-plugin-add-ons/src/api/index.js

```diff
@@ -0,0 +1,12 @@
+import { Router } from "express"
+import store from "./routes/store"
+import admin from "./routes/admin"
+
+export default (container) => {
+  const app = Router()
+
+  store(app)
+  admin(app)
+
+  return app
+}
```

Comment:
Take a look at the core routes API in `medusa-plugin-wishlist`. It basically allows you to put "extra" endpoints into one of the core routes.

Reply:
Think it would be better for the store endpoints